go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/docker/distribution v2.7.1+incompatible
 	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.7.0
-	github.com/rancher/wrangler v0.5.4-0.20200326191509-4054411d9736
+    github.com/rancher/wrangler v0.6.1
```

Comment:
Formatting looks off here. What tooling are you using? I thought that even `go mod download` or `go mod vendor` would correct this.

Reply:
i used 
- go mod download
- go mod vendor 
- go generate
and at last go fmt for the whole project

Reply:
Ywah, `go mod download` fixed the formatting for me locally.